src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -116,6 +116,8 @@ struct bdf
   get(const xrt_core::device* device, key_type)
   {
     auto pdev = get_pcidev(device);
+    if (!pdev)
+      throw xrt_core::error("Invalid device handle");
```

Comment:
Wouldn't it make more sense to nave get_pcidev throw?  There are several other calls to get_pcidev.

Reply:
Also, please use `git commit -s ...` to sign the commit.

Reply:
sure soren.